app/js/services/v3Properties.json

```diff
@@ -395,6 +395,19 @@
             "initvalue": "SELECT_EXACT_MATCH"
           }
         ]
+      },
+      "imadai": {
+        "label": "IMA-DAI",
+        "type": "featuremenu",
+        "description": "The Google IMA DAI SDK for HTML5 allows developers to make a stream request for ad and content video and get the combined (stitched) video stream in an HTML5 video environment. ",
+        "model": "config.plugins.imadai",
```

Comment:
you don't need to update the model as you just want to bundle the plugin with no config
see the receiver as an example of this

Reply:
fixed